applications/nrf_desktop/configuration/nrf5340dk_nrf5340_cpuapp/ble_scan_def.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/* This configuration file defines filters for BLE central (based on BLE
+ * names). Used only by ble_scan module.
+ */
+
+/* This structure enforces the header file is included only once in the build.
+ * Violating this requirement triggers a multiple definition error at link time.
+ */
+const struct {} ble_scan_include_once;
+
+#include "ble_event.h"
+
+static const char * const peer_name[] = {
+	[PEER_TYPE_MOUSE] = "Mouse nRF52 Desktop",
+	[PEER_TYPE_KEYBOARD] = "Keyboard nRF52 Desktop",
```

Comment:
nRF52 -> nRF53 ?

Reply:
These are used as Bluetooth device names by nRF Desktop peripherals. We need to use the same names here to let the dongle connect with the peripherals.

Reply:
We could consider changing this as a separate PR, but I am not sure if the change is actually needed.

Reply:
Okay. I understand the reasoning for that. Maybe in the future.